.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request: {}
+  workflow_dispatch: {}
+  push: 
+    branches:
+      - main
+      - master
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Is this needed if it also runs on `main`?

Reply:
It will run on main when a PR is merged and also daily. This is how we have configured it on bitbucket as well.